README.md

```diff
@@ -311,42 +311,40 @@ $ docker run -it tarantool/tarantool:2
 
 Fixed versions:
 
-| Docker tag | Dockerfile                   |
-| ---------- | ---------------------------- |
-| 1.10.0     | dockerfile/alpine_3.5_1.10.3 |
-| 1.10.1     | dockerfile/alpine_3.5_1.10.3 |
-| 1.10.2     | dockerfile/alpine_3.5_1.10.3 |
-| 1.10.3     | dockerfile/alpine_3.5_1.10.3 |
-| 1.10.4     | dockerfile/alpine_3.5_1.x    |
-| 1.10.5     | dockerfile/alpine_3.5_1.x    |
-| 1.10.6     | dockerfile/alpine_3.5_1.x    |
-| 2.1.0      | dockerfile/alpine_3.5_2.2    |
-| 2.1.1      | dockerfile/alpine_3.5_2.2    |
-| 2.1.2      | dockerfile/alpine_3.5_2.2    |
-| 2.1.3      | dockerfile/alpine_3.5_1.x    |
-| 2.1.3      | dockerfile/alpine_3.5_1.x    |
-| 2.2.0      | dockerfile/alpine_3.5_2.2    |
-| 2.2.1      | dockerfile/alpine_3.5_2.2    |
-| 2.2.2      | dockerfile/alpine_3.5_2.x    |
-| 2.2.3      | dockerfile/alpine_3.5_2.x    |
-| 2.2.2      | dockerfile/alpine_3.5_2.x    |
-| 2.3.0      | dockerfile/alpine_3.5_2.2    |
-| 2.3.1      | dockerfile/alpine_3.5_2.x    |
-| 2.3.2      | dockerfile/alpine_3.5_2.x    |
-| 2.4.0      | dockerfile/alpine_3.5_2.x    |
-| 2.4.1      | dockerfile/alpine_3.5_2.x    |
-| 2.5.0      | dockerfile/alpine_3.5_2.x    |
+| Docker tag | Dockerfile                |
+| ---------- | ------------------------- |
+| 1.10.0     | dockerfile/alpine_3.5_1.x |
+| 1.10.1     | dockerfile/alpine_3.5_1.x |
+| 1.10.2     | dockerfile/alpine_3.5_1.x |
+| 1.10.3     | dockerfile/alpine_3.5_1.x |
+| 1.10.4     | dockerfile/alpine_3.9_1.x |
+| 1.10.5     | dockerfile/alpine_3.9_1.x |
+| 1.10.6     | dockerfile/alpine_3.9_1.x |
+| 2.1.0      | dockerfile/alpine_3.5_2.x |
+| 2.1.1      | dockerfile/alpine_3.5_2.x |
+| 2.1.2      | dockerfile/alpine_3.5_2.x |
+| 2.1.3      | dockerfile/alpine_3.9_1.x |
+| 2.2.0      | dockerfile/alpine_3.5_2.x |
+| 2.2.1      | dockerfile/alpine_3.5_2.x |
+| 2.2.2      | dockerfile/alpine_3.9_2.x |
+| 2.2.3      | dockerfile/alpine_3.9_2.x |
+| 2.3.0      | dockerfile/alpine_3.5_2.x |
+| 2.3.1      | dockerfile/alpine_3.9_2.x |
+| 2.3.2      | dockerfile/alpine_3.9_2.x |
+| 2.4.0      | dockerfile/alpine_3.9_2.x |
+| 2.4.1      | dockerfile/alpine_3.9_2.x |
+| 2.5.0      | dockerfile/alpine_3.9_2.x |
 
 Rolling versions:
 
 | Docker tag | Dockerfile                |
 | ---------- | ------------------------- |
-| 1          | dockerfile/alpine_3.5_1.x |
-| 2.1        | dockerfile/alpine_3.5_2.2 |
-| 2.2        | dockerfile/alpine_3.5_2.x |
-| 2.3        | dockerfile/alpine_3.5_2.x |
-| 2.4        | dockerfile/alpine_3.5_2.x |
-| 2/latest   | dockerfile/alpine_3.5_2.x |
+| 1          | dockerfile/alpine_3.9_1.x |
+| 2.1        | dockerfile/alpine_3.9_1.x |
+| 2.2        | dockerfile/alpine_3.9_2.x |
+| 2.3        | dockerfile/alpine_3.9_2.x |
+| 2.4        | dockerfile/alpine_3.9_2.x |
+| 2/latest   | dockerfile/alpine_3.9_2.x |
```

Comment:
Raw idea: Maybe update only rolling versions for now, but add future fixed ones based on Alpine 3.9/3.12? This way looks more safe for me.

Reply:
Alpine 3.9 is the oldest release version mentioned as supported on alpine site, so later will update only rolling versions, but for now we need to update it ones.

Reply:
But some fixed versions are on Alpine 3.5 anyway, so 'we cannot offer images based on an unsupported Apline version' would be too pathetic.

Up to you, but I still think that bump it in the rolling and future fixed versions is more safe way.